src/data/finsupp/basic.lean

```diff
@@ -1134,10 +1141,24 @@ lemma update_eq_sub_add_single [add_group G] (f : α →₀ G) (a : α) (b : G)
   f.update a b = f - single a (f a) + single a b :=
 by rw [update_eq_erase_add_single, erase_eq_sub_single]
 
+/-- Given a family of functions `f i : α → N` indexed over `S : finset ι`, the sum of this family
+  over `S` is a function `α → N` whose value at `p : α` is `∑ (i : ι) in S, (f i) p` -/
+lemma finset_sum_apply [add_comm_monoid N] (S : finset ι) (a : α) (f : ι → α →₀ N) :
+  (S.sum f) a = ∑ (i : ι) in S, (f i) a :=
```

Comment:
A little easier to read as
```suggestion
  (∑ i in S, f i) a = ∑ i in S, f i a :=
```
so as not to confuse `finset.sum` with `finsupp.sum`